configuration/pegasus_jaas.conf

```diff
@@ -0,0 +1,10 @@
+client {
+	com.sun.security.auth.module.Krb5LoginModule required
```

Comment:
Does this config format support comment? Can you write a comment of description on top of each config item?
Furthermore, does it mean that every of our users should have this file in their project?

Reply:
1.I have tested that this config does't support comment. So I can write a file later to show what`s the meaning of these config item.
2. Yes, our users should have this file in their project.

Reply:
Yes, every our users should have this file in their project

Reply:
I removed the `pegasus_jaas.conf`. So we can add it in later pull request. And I will refactor it in a better way at that time.